spec/asyncapi.md

```diff
@@ -84,20 +84,23 @@ It means that the [application](#definitionsApplication) allows [consumers](#def
 
 ## <a name="definitions"></a>Definitions
 
+#### <a name="definitionsServer"></a>Server
+A server MAY be a message broker that capable to sending and/or receiving between [producer](#definitionsProducer) and [consumer](#definitionsConsumer) but also a server MAY be a service with WebSocket API that enables message-drive communication between browser-to-server or server-to-server.
```

Comment:
```suggestion
A server may be a message broker that is capable of sending and/or receiving between a [producer](#definitionsProducer) and [consumer](#definitionsConsumer). A server may also be a service with WebSocket API that enables message-drive communication between browser-to-server or server-to-server.
```

caps for no reason are odd, so let's remove them ❤️😀👍🏽

Reply:
about 'MAY' word with capitalization I found it on here <img width="750" alt="image" src="https://user-images.githubusercontent.com/20577131/163903417-5ba2129e-1372-4598-9f12-0b4c3f6e936e.png">
might be you want check this out , it should refer to interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt), right?

Reply:
hmmm... ok, let's have @derberg confirm the `MAY` question, but go ahead w/ the other changes 👍🏽

Reply:
Alright 👍🏽